src/entries/popup/hooks/useTokenDetailsShortcuts.ts

```diff
@@ -0,0 +1,68 @@
+import { useCallback } from 'react';
+
+import { shortcuts } from '~/core/references/shortcuts';
+import { useContainerRef } from '~/design-system/components/AnimatedRoute/AnimatedRoute';
+
+import { simulateClick } from '../utils/simulateClick';
+
+import useKeyboardAnalytics from './useKeyboardAnalytics';
+import { useKeyboardShortcut } from './useKeyboardShortcut';
+
+interface UseTokenDetailsShortcutsParameters {
+  getTokenExist: () => boolean;
+  toggleHideToken: () => void;
+  togglePinToken: () => void;
+  copyTokenAddress: () => void;
+}
+
+export function useTokenDetailsShortcuts({
+  getTokenExist,
+  togglePinToken,
+  toggleHideToken,
+  copyTokenAddress,
+}: UseTokenDetailsShortcutsParameters) {
+  const { trackShortcut } = useKeyboardAnalytics();
+
+  const containerRef = useContainerRef();
+
```

Comment:
We technically could use `useTokensShortcuts`, but would add a few if/else logics alongside with custom logic which would make the hook not seem very clean + it's only used on the homepage for token list anyways. 

Instead we can create a hook for the detail page separately.